### PR TITLE
View Activities List

### PR DESCRIPTION
**Pull request by darrenlittlejohn**
Added activities component.

### CODE REVIEW

File: front-end/src/index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
+import Activities from './pages/Activities'
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(<Activities />, document.getElementById('root'));
```
Comment (orangewolf) [inline front-end/src/index.js]: this means you are no longer using App as your root and so no other items are possible.  you should instead import Activities in to App and display the Activities as a child component of App

File: front-end/src/pages/Activities.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+
+
+import React, { Component } from 'react';
+
+//Activities List
+class ActivitiesList extends Component {
```
Comment (orangewolf) [inline front-end/src/pages/Activities.js]: Is it Activities or ActivitiesList?  file name should match class should match export

**Comment by orangewolf** [comment]
@darrenlittlejohn can you hear me now in slack?

File: front-end/src/index.js (hunk 1 above)
Comment (darrenlittlejohn) [reply front-end/src/index.js]: Got it.

**Pull request merged**